### PR TITLE
New properties: SVG rx and ry

### PR DESCRIPTION
**Pull request by estelle**
related to https://github.com/mdn/content/issues/34763

### CODE REVIEW

File: files/en-us/web/css/rx/index.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+```css
+svg {
+  border: 1px solid;
+  width: 300px;
+}
+
+ellipse:first-of-type {
+  rx: 30;
+  fill: magenta;
```
Comment (chrisdavidmills) [inline files/en-us/web/css/rx/index.md:225]: Interesting; in Firefox, you don't see the colored ellipses rendered; you do in Chrome. Worth mentioning at all?

File: files/en-us/web/css/ry/index.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+{{Specifications}}
+
+## Browser compatibility
+
+{{Compat}}
+
+## See also
+
+- Geometrx properties: `ry`, {{cssxref("cx")}}, {{cssxref("cy")}}, {{cssxref("r")}}, {{cssxref("rx")}}, {{cssxref("x")}}, {{cssxref("y")}}, {{cssxref("height")}}, {{cssxref("height")}}
```
Comment (chrisdavidmills) [inline files/en-us/web/css/ry/index.md:250]:
```suggestion
- Geometry properties: `ry`, {{cssxref("cx")}}, {{cssxref("cy")}}, {{cssxref("r")}}, {{cssxref("rx")}}, {{cssxref("x")}}, {{cssxref("y")}}, {{cssxref("height")}}, {{cssxref("height")}}
```

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle nice work as always; couple of bits that need looking at here. 

Also, most of the comments I made on the `rx` page apply to the `ry` page too. Have a look at both as you go through and make the changes.

I made a couple of additional comments on the `ry` page.

File: files/en-us/web/css/ry/index.md (hunk 2 above)
Comment (estelle) [reply files/en-us/web/css/ry/index.md:250]: LOL. oops, got overzealous with  replacing ry with rx

File: files/en-us/web/css/rx/index.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
+---
+title: rx
+slug: Web/CSS/rx
+page-type: css-property
+browser-compat: css.properties.rx
+---
+
+{{CSSRef}}
+
+The **`rx`** [CSS](/en-US/docs/Web/CSS) property defines the x-axis, or horizontal, radius of an SVG {{SVGElement("ellipse")}} and the horizontal curve of the corners of the SVG {{SVGElement("rect")}} rectangle. This property can only be used with the SVG `<ellipse>` and `<rect>` elements. If present, it overrides the shape's {{SVGAttr("rx")}} attribute.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md]: kept the note below and removed the sentence instead.

File: files/en-us/web/css/rx/index.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+
+> [!NOTE]
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md:12]: kept note, removing the sentence from the property summary first paragraph

File: files/en-us/web/css/rx/index.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+> The `rx` property only applies to {{SVGElement("ellipse")}} and {{SVGElement("rect")}} elements nested in an {{SVGElement("svg")}}. It doesn't apply to other SVG elements or HTML elements or pseudo-elements.
+
+## Syntax
+
+```css
+/* initial value */
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md]: updated

File: files/en-us/web/css/rx/index.md [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+rx: auto;
+
+/* length and percentage values */
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md]: updated

File: files/en-us/web/css/rx/index.md [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+rx: inherit;
+rx: initial;
+rx: revert;
+rx: revert-layer;
+rx: unset;
+```
+
+### Values
+
+The {{cssxref("length-percentage")}} property value denotes the horizontal radius of ellipses and the horizontal border-radius of rectangles.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md:36]: changed to:

> The {{cssxref("length")}}, {{cssxref("percentage")}}, or `auto` keyword value denotes ....

File: files/en-us/web/css/rx/index.md [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+
+- {{cssxref("length")}}
+
+  - : As an absolute or relative length, it can be expressed in any unit allowed by the CSS {{cssxref("&lt;length&gt;")}} data type. Unitless and `px` values are SVG coordinate units if the {{SVGAttr("viewBox")}} attribute is included; otherwise, CSS pixel units are used. Negative values are invalid.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md]: updated

File: files/en-us/web/css/rx/index.md [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+
+#### Results
+
+{{EmbedLiveSample("Ellipse horizontal radius percentage values", "300", "180")}}
+
+When using percentage values for `rx`, the values are relative to the width of the SVG viewport. Here, the size of the styled ellipse horizontal radius is `30%` of the width of the current SVG viewport. As the width defaulted to `300px`, the `rx` value is `90px`.
+
+### Using unitless values
+
+This example demonstrates the effects of the viewBox attribute on unitless `rx` value declaration.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/rx/index.md]: updated

File: files/en-us/web/css/ry/index.md [hunk 10]
```diff
@@ -0,0 +1,256 @@
...
+
+## Examples
+
+### Creating rounded corners
+
+This example demonstrates creating rectangles with rounded corners by applying the `ry` property to SVG `<rect>` elements.
+
+#### HTML
+
+We include an SVG image with four `<rect>` elements; all the rectangles are the same except for their {{SVGAttr("x")}} attribute value, which positions them along the y-axis.
```
[the remark this answers is not in the document]
Comment (estelle) [reply files/en-us/web/css/ry/index.md]: good catch. updated.

**Review by estelle — COMMENTED** [review]
Thanks @chrisdavidmills

File: files/en-us/web/css/rx/index.md (hunk 1 above)
Comment (estelle) [reply files/en-us/web/css/rx/index.md:225]: this seems to be FF not supporting pseudo-classes in SVGs. That's a bug. diving deeper.

File: files/en-us/web/css/rx/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/rx/index.md]: Looks fine.

File: files/en-us/web/css/rx/index.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply files/en-us/web/css/rx/index.md]: This works well.

**Review by chrisdavidmills — APPROVED** [review]
Superb work @estelle. This all looks great. and I agree with removing the example based on the buggy/inconsistent feature for now. It would confuse several people. And we have enough examples to be going on with.

**Pull request merged**